### DISCUSSION TITLE
ZepetoPlayers 내부의 Layer 설정

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#377] [5. Q&A]
ZepetoPlayers 안에 존재하는 Player 설정에 Layer를 따로 고를 수가 있는데.
이는 어디다 쓰이는건가요?

캐릭터가 생성될때 레이어를 설정해주는건가 해서
제가 따로 레이어를 생성해서 넣어보니 A game object can only be in one layer. The layer needs to be in the range [0...31] 
라는 오류가 뜹니다.

**Comment by moondory77-naverz** [comment]
안녕하세요.
말씀주신대로, ZepetoPlayers -> Player -> Layer 옵션은
생성되는 캐릭터 인스턴스의 Layer를 설정하는 기능이 맞습니다.

현재 해당 옵션 설정에 오류가 있음을 확인했으며, 다음 업데이트를 통해 바로 수정드리도록 하겠습니다.
불편을 드려 죄송합니다.
